src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -289,6 +348,33 @@ public string Format(string rawXmlText, Compilation compilation = null)
             return state.GetText();
         }
 
+        public IEnumerable<TaggedText> Format(ISymbol symbol, SemanticModel semanticModel, int position, CancellationToken cancellationToken, SymbolDisplayFormat format = null)
+        {
+            var docSymbol = symbol;
+            if (symbol.GetTypeParameters().Any())
+            {
+                docSymbol = symbol.OriginalDefinition;
+            }
+            var rawXmlText = GetDocumentation(docSymbol, semanticModel.Compilation, cancellationToken);
```

Comment:
```suggestion
            var rawXmlText = GetDocumentation(symbol.OriginalDefinition, semanticModel.Compilation, cancellationToken);
```

Reply:
i'm not sure here. basically, we need the special handling for type parameters only if there are any.
the special handling means: use the originaldefinition to get the documentation - right?

Reply:
the code i wrote should be correct here.  we should always go back to the original-def to get teh documentation.